bindata/network/ovn-kubernetes/managed/alert-rules-control-plane.yaml

```diff
@@ -139,3 +139,47 @@ spec:
       for: 25m
       labels:
         severity: warning
+    - alert: OVNKubernetesNorthboundDatabaseLeaderError
+      annotations:
+        summary: OVN northbound database(s) have no RAFT leader
+        description: OVN northbound database(s) have no RAFT leader. Networking control plane is degraded.
+      expr: |
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        count(max_over_time(ovn_db_cluster_server_role{db_name="OVN_Northbound", server_role="leader"}[5m])) == 0
+      for: 15m
```

Comment:
It whens if the promql expression is true for 15 minutes it will fire an alert.
Because the promql expression looks back 5 minutes, it means the total time to fire will be 20 minutes of continuous time with no leader seen.
I picked this time to detect permanently broken leader election. 
Do you want to reduce it? It does seem too long on second thought.

Reply:
I would say if this is true for even over a minute it should alert. If there is no leader then it represents some kind of outage or problem between raft nodes that should never happen. I think under normal conditions, raft leadership will always be transferred, or if an election timer goes off it should take only a few seconds max to find a new leader. I'll defer to @igsilya to what he thinks would be a reasonable value here.

Reply:
I reduced it to five minutes. We have to consider data is scared every 30s and evaluated every 1m. I am uncomfortable having an alert lower than 5 minutes for this without a lot of testing.